lib/src/Uncertainty/Model/openturns/DistributionImplementation.hxx

```diff
@@ -1635,6 +1636,7 @@ protected:
     Point operator() (const Point & point) const override
     {
       const Scalar logPDF = p_distribution_->computeLogPDF(point);
+      if (logPDF == SpecFunc::LowestScalar) return Point(1, 0.0);
```

Comment:
I cannot understand why this code is necessary. If std::exp() handles -INF, then -exp(-INF) should be -0 and multiplied by a negative number (since logPDF must be negative here), this should give a +0. So your code suggests that std::exp(-INF) does not produce +0: what does it produce then?

Reply:
The problem is the line below: https://github.com/openturns/openturns/blob/a18c583c8a35239d8db361dd77178b9c87f1836d/lib/src/Uncertainty/Model/openturns/DistributionImplementation.hxx#L1640

Without the check, we would get the indeterminate form:
```
-std::exp(-INF) * (-INF) = (-0) * (-INF) = NAN
```
Of course, we know that the limit of x log(x) is 0 when x goes to 0, but the arithmetic of floats does not.